### DISCUSSION TITLE
Responding to a takenote impersonator

### DISCUSSION THREAD

**Discussion by binyamin** [taniarascia/takenote#499] [Q&A]
I received this email today. I'd like to respond with something snarky. Any ideas? Just FYI, it's addressed to my dog (my name is binyamin).
> Dear Wilson,
>
> I saw that you liked Take Note on GitHub. The team behind Take Note hasn’t created a hosted version because it’s really a lot of work. Nonetheless, Take Note is one of the first apps launched on a project I’m working on called Deta Space: a ‘personal cloud’ where the app, notes, and data are online and under your control.
>
> I was wondering if you’d be interested in trying an early version of Deta Space and giving me your feedback? I’d love to hear what you think.
>
> Thank you already for your time!
>
> Cheers,
> Max

**Comment by kodyclemens** [comment]
Hey binyamin,

This project is OSS under the MIT license so folks can basically do whatever they would like with the project. I agree that the email request does seem odd though.
